defaults/cslc_s1.yaml

```diff
@@ -0,0 +1,34 @@
+runconfig:
+   name: cslc_s1_workflow_default
+
+   groups:
+       pge_name_group:
+           pge_name: CSLC_S1_PGE
+
+       input_file_group:
+           # Required. List of reference and secondary S1 ZIP files
```

Comment:
do we really need both reference and secondary zip files to coregister every burst? I think it depends on the design. Since we are at the very beginning I suggest that we avoid requiring the reference zip file every time we coregister a secondary burst to a reference burst.